gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/services/CgmesExportService.java

```diff
@@ -55,11 +55,13 @@ public class CgmesExportService {
     private static final Logger LOGGER = LoggerFactory.getLogger(CgmesExportService.class);
     private static final double DEFAULT_P_TOLERANCE = 10;
     private static final DateTimeFormatter CGMES_FORMATTER = DateTimeFormatter.ofPattern("yyyyMMdd'T'HHmm'Z'_'[process]_[tso]_[type]_001.xml'");
+    public static final String MODELING_AUTHORITY_DEFAULT_VALUE = "https://farao-community.github.io/";
     private final Logger businessLogger;
     private final FileExporter fileExporter;
 
     private final FileImporter fileImporter;
     private final UrlValidationService urlValidationService;
+    private final Map<String, String> modelingAuthorityMap;
```

Comment:
Your class is a Service class, treated as a singleton by default in Spring. It should be stateless, so adding an instance attribute storing _data_ (as opposed to _Spring beans_) is not the best way to do. You apparently only use the modeling authority map inside of the `updateModelAuthorityParameter` function. So why don't you just define a local variable there and call the `processConfiguration.getModelingAuthoritySet()` function to retrieve your data?